android/app/fabric.properties

```diff
@@ -0,0 +1,3 @@
+#Contains API Secret used to validate your application. Commit to internal source control; avoid making secret public.
+#Sun Dec 13 13:22:06 PST 2015
+apiSecret=83176563693e1eefa1a91359c4151b88c3fd17e52ddf6d64416b7f1e0fa9cede
```

Comment:
Take this out. Not really even sure what this is used for but its bad practice to include this in the repository.

Reply:
removed